mis_builder/models/aep.py

```diff
@@ -28,6 +28,13 @@ class AccountingExpressionProcessor(object):
         * accounts is a list of accounts, possibly containing % wildcards
         * an optional domain on move lines allowing filters on eg analytic
           accounts or journal
+        * an optional parameter for rate calculation. this parameter will
+          override the default value set in the report instance.
+          possible values are
+          empty or now: current date exchange ratenow,
+          start: start date of period (date_from)
+          end: end date of period (date_to)
+          daily: daily rate of move line date.
```

Comment:
Could you explain your idea?
Do you think of adding a method for rate calculation directly in account.move.line?